### ISSUE TITLE
Add support for notification types beyond email

### ISSUE THREAD

**Issue by icco** [SumoLogic/sumologic-kubernetes-collection#3079]
**Is your feature request related to a problem? Please describe.** 

I would like to be able to channel the monitors generated by this to something besides email, such as Slack for warnings and OpsGenie for Criticals.

**Describe the solution you'd like** A clear and concise description of what you want to happen.

Being able to pass in an entire notification json blob for each level would be my preferred method.

**Describe alternatives you've considered** 

We've thought about just importing the terraform used in this module.

**Additional context** 

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/9ceeaf0efe46d420389e08a388d93f068b83c554/deploy/helm/sumologic/conf/setup/monitors.sh#L71-L94 is where these notifications are configured.

**Comment by kkujawa-sumo** [comment]
As mentioned [here](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/monitor) sumologic supports for `connection_type` following options: `Email`, `Webhook`, `PagerDuty`,through `Webhook` supports several integrations (details [here](https://registry.terraform.io/providers/SumoLogic/sumologic/latest/docs/resources/connection)): 
`AWSLambda`, `Azure`, `Datadog`, `HipChat`, `Jira`, `PagerDuty`, `Slack`, `Webhook`, `NewRelic`, `MicrosoftTeams`, `ServiceNow`, and `SumoCloudSOAR` so we could consider adding support for other notification types

**Comment by icco** [comment]
Yeah, the underlying terraform supports that, I just want support in the helm chart for it.
